AABB_tree/include/CGAL/internal/AABB_tree/AABB_search_tree.h

```diff
@@ -83,7 +85,7 @@ namespace CGAL
                 typedef typename CGAL::Orthogonal_k_neighbor_search<TreeTraits> Neighbor_search;
                 typedef typename Neighbor_search::Tree Tree;
         private:
-                Tree* m_p_tree;
+                std::unique_ptr<Tree> m_p_tree;
```

Comment:
I wonder if we really need yet another pointer here (rather than a plain `Tree` object)

Reply:
I don't think using a pointer is really necessary. Will change it.